### PR TITLE
Update to IML 5 repo 

### PR DESCRIPTION
**Pull request by jgrund**
Update to the IML 5 Copr repo.

In addtion remove module-tools and use the Docker builder.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: .copr/Makefile [hunk 1]
```diff
@@ -24,3 +33,33 @@ srpm:
...
+		${TMPDIR}/_topdir/SOURCES
+
+	cd ${TMPDIR}/scratch; \
+	python setup.py sdist -d ${TMPDIR}/_topdir/SOURCES/
+	
+	rpmbuild -bs -D "_topdir ${TMPDIR}/_topdir" ${TMPDIR}/_topdir/SPECS/python-iml-manager.spec
+
+	cp -rf ${TMPDIR}/_topdir ${BUILDROOT}/
+	cp -r ${TMPDIR}/_topdir/SRPMS/* $(outdir)
+	rm -rf ${TMPDIR}
```
Comment (johnsonw) [inline .copr/Makefile]: Is this remove necessary?

File: .travis.yml [hunk 2]
```diff
@@ -61,13 +61,22 @@ jobs:
         - pip install black
       script:
         - black --check ./
+    - stage: test
+      name: "copr build test"
+      script:
+        - export SPEC=python-iml-manager.spec
+        - export SRPM_TASK=iml-srpm
```
Comment (johnsonw) [inline .travis.yml]: What does the SRPM_TASK variable indicate?

File: .copr/Makefile (hunk 1 above)
Comment (jgrund) [reply .copr/Makefile]: No, but it doesn't hurt.

File: .travis.yml (hunk 2 above)
Comment (jgrund) [reply .travis.yml]: Added in a PR for the Dockerfiles. 

It allows the user to override what make task is used to produce the srpm.

**Pull request merged**